extensions/azdata/src/extension.ts

```diff
@@ -4,13 +4,25 @@
  *--------------------------------------------------------------------------------------------*/
 
 import * as azdataExt from 'azdata-ext';
-import { findAzdata, IAzdataTool, manuallyInstallOrUpgradeAzdata } from './azdata';
+import * as vscode from 'vscode';
+import * as constants from './constants';
+import { findAzdata, IAzdataTool, manuallyInstallOrUpgradeAzdata, promptForEula } from './azdata';
 import * as loc from './localizedConstants';
+import Logger from './common/logger';
 
 let localAzdata: IAzdataTool | undefined = undefined;
+let eulaAccepted: boolean = false;
 
-export async function activate(): Promise<azdataExt.IExtension> {
+export async function activate(context: vscode.ExtensionContext): Promise<azdataExt.IExtension> {
 	localAzdata = await checkForAzdata();
+	eulaAccepted = !!context.globalState.get(constants.acceptEula);
```

Comment:
You don't have to !! this - if it's not set it'll be undefined and act in the same way.

Reply:
It is needed for the case when acceptEula was previously accepted by the user.

Reply:
No because it'll be `true` when that happens - you don't need to convert it into a boolean since it already will be one.

Reply:
Will add a comment to make that clearer

Reply:
Ok...but I don't really see the point. That's standard behavior for this and I think the var names make it clear enough.